apps/wallet/src/ui/app/pages/home/tokens/TokensPage.module.scss

```diff
@@ -103,3 +103,22 @@
     position: sticky;
     bottom: 0;
 }
+
+.activity {
+    margin-top: 10px;
+}
+
+.details-page {
+    overflow-y: hidden;
+}
+
+.tx-content {
+    height: 210px;
```

Comment:
I think we should make this auto expand to the remaining height of the `.container` (I think adding `flex:1` here and the `.container` should do it and use [override-main-padding](https://github.com/MystenLabs/sui/blob/18563ec89fcab0a7b39fc0011be17b759a1a3afb/apps/wallet/src/ui/styles/utils/index.scss#L24) for spacing